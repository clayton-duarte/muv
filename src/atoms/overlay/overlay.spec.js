import React from 'react';

import Overlay from '.';

it('should render a overlay', () => {
  const wrapper = mount(
    <Overlay />,
  );
  expect(wrapper).toMatchSnapshot();
});
