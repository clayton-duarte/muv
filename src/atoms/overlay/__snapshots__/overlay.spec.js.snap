// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a overlay 1`] = `
<Overlay
  visible={false}
>
  <styled.aside
    theme={
      Object {
        "animation": Object {
          "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
          "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
          "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
          "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
          "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
        },
        "color": Object {
          "black": "#222222",
          "darkGray": "#515151",
          "error": "#dd3333",
          "gray": "#a9abae",
          "lightGray": "#e5e5e5",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0081c8",
          "secondary": "#00245f",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "font": Object {
          "color": "#222222",
          "family": "\\"Ubuntu\\", sans-serif",
          "size": "16px",
          "spacing": 0,
        },
        "shape": Object {
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "maxWidth": "1280px",
          "padding": ".75rem .875rem",
          "radius": ".5rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          "size": "16px",
        },
        "transition": Object {
          "time": ".3s ease",
        },
      }
    }
    visible={false}
  >
    <StyledComponent
      forwardedClass={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "lastClassName": "ihgAJq",
            "rules": Array [
              "
pointer-events: ",
              [Function],
              ";
transition: ",
              [Function],
              ";
background: ",
              [Function],
              ";
z-index: ",
              [Function],
              ";
opacity: ",
              [Function],
              ";
justify-content: center;
align-items: center;
position: fixed;
padding: 1rem;
display: flex;
bottom: 0;
right: 0;
left: 0;
top: 0;
",
            ],
          },
          "defaultProps": Object {
            "theme": Object {
              "animation": Object {
                "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
              },
              "color": Object {
                "black": "#222222",
                "darkGray": "#515151",
                "error": "#dd3333",
                "gray": "#a9abae",
                "lightGray": "#e5e5e5",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#0081c8",
                "secondary": "#00245f",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "font": Object {
                "color": "#222222",
                "family": "\\"Ubuntu\\", sans-serif",
                "size": "16px",
                "spacing": 0,
              },
              "shape": Object {
                "border": ".125rem solid rgba(0, 0, 0, .15)",
                "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "maxWidth": "1280px",
                "padding": ".75rem .875rem",
                "radius": ".5rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "16px",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            },
          },
          "displayName": "styled.aside",
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "aside",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      theme={
        Object {
          "animation": Object {
            "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
            "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
            "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
            "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
            "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
          },
          "color": Object {
            "black": "#222222",
            "darkGray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightGray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0081c8",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "\\"Ubuntu\\", sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "maxWidth": "1280px",
            "padding": ".75rem .875rem",
            "radius": ".5rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
      visible={false}
    >
      <aside
        className="sc-bdVaJa ihgAJq"
      />
    </StyledComponent>
  </styled.aside>
</Overlay>
`;
